src/site/content/en/blog/reporting-api/index.md

```diff
@@ -0,0 +1,664 @@
+endpoint you've specified.
+
+This API enables you to activate a reporting mechanism that's **operated by the browser**.
+When the Reporting API is configured on your site, **reports can be generated** when certain
+policies are violated (or when deprecated APIs are used, when the browser intervenes on
+your site or crashes, and more). The browser then sends those report(s) to the
+**endpoint(s)**, that is report collectors, you've configured.
+
+Setting up the Reporting API gives you peace of mind that when users experience these
+types of unforeseen errors, you'll be aware and able to fix them if needed. ✨
+
+```http
+  Reporting-Endpoints: main-endpoint="https://reports.example/main", default="https://reports.example/default"
+  Content-Security-Policy: script-src 'self'; object-src 'none'; report-to main-endpoint;
+```
+
+This post covers what this API can do, how it works, and how you can use it.
+
+Let's dive in!
+
+## Demo and code
+
+💥 [Demo site: report generation page](https://reporting-api-demo.glitch.me). This page
+uses the new Reporting API via the `Reporting-Endpoints` header. See the
+[code](https://glitch.com/edit/#!/reporting-api-demo).
+
+📬 [Demo site: reporting endpoint](https://reports-endpoint.glitch.me). This server
+receives and displays reports. See the
+[code](https://glitch.com/edit/#!/reports-endpoint).
+
+## Use cases and report types
+
+{% Banner 'caution', 'body' %}
+
+[Network Error Logging](https://w3c.github.io/network-error-logging/) isn't listed because it isn't supported in the new version of
+the API. Check the [migration guide](/reporting-api-migration/#network-error-logging) for details.
+
+{% endBanner %}
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>Report type</th>
+        <th>Example of a situation where a report would be generated</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP">CSP</a> violation (Level 3 only)</td>
+        <td>You've set a <code>Content-Security-Policy</code> on your site, but one of your pages is trying to load a script that's not allowed by your CSP.</td>
+      </tr>
+      <tr>
+        <td><a href="/why-coop-coep/#coop">COOP</a> violation</td>
+        <td>You've set a <code>Cross-Origin-Opener-Policy</code> on your site, but a cross-origin window is trying to interact directly with your document.</td>
+      </tr>
+      <tr>
+        <td><a href="/why-coop-coep/#coep">COEP</a> violation</td>
+        <td>You've set a <code>Cross-Origin-Embedder-Policy</code> on your site, but a page on your site includes a cross-origin iframe that has not opted into being loaded by cross-origin documents.</td>
+      </tr>
+      <tr>
+        <td>Document Policy violation</td>
+        <td>Your site has a document policy that prevents usage of <code>document.write</code>, but a script on your page tries to use <code>document.write</code>.</td>
```

Comment:
`Your site has a document policy that prevents usage of <code>document.write</code>, but a script on your page tries to use <code>document.write</code>.`
```suggestion
        <td>The page has a document policy that prevents usage of <code>document.write</code>, but a script tries to use <code>document.write</code>.</td>
```